src/main/java/racingcar/model/Winners.java

```diff
@@ -0,0 +1,34 @@
+package racingcar.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Winners {
+
+    private List<Car> raceReserts;
```

Comment:
```suggestion
    private final List<Car> raceReserts;
```

여기도 final 키워드를 사용해서 값 변화를 막아주세요 ″̮